hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,37 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset in state, this field MUST NOT be set.<br/>
+     * If unset in this transaction and set in state, the current value SHALL
+     *  be removed.<br/>
+     * If set in this transaction, the existing key MUST sign this transaction.
```

Comment:
The new key signature requirement probably should also be specified.
```suggestion
     * If this field is set, the existing key MUST sign this transaction.
     * If this field is set, the new fee schedule key MUST sign this transaction.
```

Reply:
Fixed